lib/endpoints/class-wp-rest-users-controller.php

```diff
@@ -517,6 +517,15 @@ public function prepare_item_for_response( $user, $request ) {
 		}
 
 		$context = ! empty( $request['context'] ) ? $request['context'] : 'embed';
+
+		if ( isset( $data['capabilities'] ) && empty( $data['capabilities'] ) ) {
```

Comment:
Not sure what I was thinking exactly, thanks for catching that and pointing it out.